requirements.txt

```diff
@@ -4,7 +4,7 @@ numpy>=1.16.6
 torch>=1.4
 future>=0.17.1  # required for builtins in setup.py
 # pyyaml>=3.13
-PyYAML>=5.1, !=5.4.*  # OmegaConf requirement >=5.1
+PyYAML>=5.1  # OmegaConf requirement >=5.1
```

Comment:
OmegaConf declares it's version dependency on pyyaml so this line should no longer been needed (At some point in the far past it didn't, which was the reason for adding this in the first place).

https://github.com/omry/omegaconf/blob/master/requirements/base.txt